tftpsync/susemanager-tftpsync-recv/susemanager-tftpsync-recv.spec

```diff
@@ -83,10 +83,10 @@ if [ -f /etc/apache2/conf.d/susemanager-tftpsync-recv.conf.rpmnew ]; then
             exit 1;
     fi;
 
-    SUMA_IP=$( host $PARENT_FQDN | awk '{ print $4 }' )
+    SUMA_IP=$(getent hosts $PARENT_FQDN | awk '{ print $1 }' || echo "")
```

Comment:
This is not needed anymore.

Reply:
I have reverted the host check. So this comment is not needed anymore.